compliance_checker/cf/cf.py

```diff
@@ -1893,14 +1894,12 @@ def check_calendar(self, ds):
                 continue
```

Comment:
Actually the easiest way to fix everything is to change the for loop to iterate over

```python
for timevar in ds.get_variables_by_attributes(calendar=lambda x : x is not None):
```